### PR TITLE
Add a setup.py

### PR DESCRIPTION
**Pull request by daniviga**
This allows an easy installation with pip and setuptools.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+README = """
+Python and OpenQuake-based Toolkit for Analysis of Strong Motions and
+Interpretation of GMPEs.
+Copyright (C) 2014-2017 GEM Foundation
+"""
+
+setup(
+    name='hmtk',
+    version='0.0.1',
```
Comment (daniviga) [inline setup.py]: Which version should I put here?

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+url = "https://github.com/GEMScienceTools/gmpe-smtk"
+
+README = """
+Python and OpenQuake-based Toolkit for Analysis of Strong Motions and
+Interpretation of GMPEs.
+Copyright (C) 2014-2017 GEM Foundation
+"""
+
+setup(
+    name='smtk',
```
Comment (daniviga) [inline setup.py]: We have a miss-aligment between the package name `smtk` and the app name `gmpe-smtk`

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    version='0.0.1',
```
Comment (daniviga) [inline setup.py]: Which version we want to put here?

Comment (g-weatherill) [inline setup.py]: Lets go with 0.1.0, thanks

**Comment by daniviga** [comment]
Merging because I need this for the new VM

**Pull request merged**